it-it/coffeescript-it.html.markdown

```diff
@@ -4,6 +4,8 @@ contributors:
   - ["Luca 'Kino' Maroni", "http://github.com/kino90"]
   - ["Tenor Biel", "http://github.com/L8D"]
   - ["Xavier Yao", "http://github.com/xavieryao"]
+translators:
+    - ["Tommaso Pifferi","http://github.com/neslinesli93"]
```

Comment:
I forgot to add myself as a translator in coffeescript-it in a previous commit:
1aaf79b6e56b7d4947335dd60613f8096b66b0d9

This PR was just about elixit, but I committed the change about coffeescript as well